isthmus/src/main/java/io/substrait/isthmus/expression/LiteralConverter.java

```diff
@@ -170,19 +169,15 @@ public Expression.Literal convert(RexLiteral literal) {
           INTERVAL_MINUTE,
           INTERVAL_MINUTE_SECOND,
           INTERVAL_SECOND -> {
-        // we need to convert to microseconds.
-        // TODO: don't need to anymore
-        int scale = literal.getType().getScale();
-        var intervalLength = literal.getValueAs(BigDecimal.class).longValue();
-        var adjustedLength =
-            scale > 6
-                ? intervalLength / ((int) Math.pow(10, scale - 6))
-                : intervalLength * ((int) Math.pow(10, 6 - scale));
-        var days = adjustedLength / MICROS_IN_DAY;
-        var totalMicroseconds = adjustedLength - days * MICROS_IN_DAY;
-        var seconds = totalMicroseconds / 1_000_000;
-        var microseconds = totalMicroseconds - 1_000_000 * seconds;
-        yield intervalDay(n, (int) days, (int) seconds, microseconds, 6 /* micros */);
+        // Calcite represents day/time intervals in milliseconds, despite a default scale of 6.
+        var totalMillis = Objects.requireNonNull(literal.getValueAs(Long.class));
```

Comment:
so is the case here that Calcite has _always_
- literal.getType().getScale() = 6
- literal.getValue() returns millis?

Reply:
Ah, you actually already answered that in the PR description:
> Despite all day/time interval RexLiteral created from SQL by Calcite having a (default) scale of 6, their value is always in milliseconds. 

👌